HandleNHNotificationCall/function.json

```diff
@@ -7,10 +7,26 @@
       "type": "queueTrigger",
       "direction": "in"
     },
+    {
+      "direction": "in",
+      "name": "betaTestUser",
+      "type": "table",
+      "tableName": "%BETA_USERS_TABLE_NAME%",
+      "connection": "BETA_USERS_STORAGE_CONNECTION_STRING",
+      "filter": "RowKey eq '{installationId}'",
```

Comment:
Not sure this is gonna work in this context. How did you test it?

Reply:
I do a system test pointing directly to an Azure Storage: assuming that the message in the queue is a Json containing the filed installationId in the root, it's work fine.